modules/user_data/variables.tf

```diff
@@ -18,11 +23,23 @@ variable "device_name" {
 variable "backup_schedule" {
   description = "Cron expression for the backup job."
   type        = string
+
+  #TO BE DONE SOON - Doesn't work yet
```

Comment:
Remove this if it's not going to be implemented.

Reply:
Removed.